types/types.go

```diff
@@ -462,8 +462,10 @@ type SystemContext struct {
 	RegistriesDirPath string
 	// Path to the system-wide registries configuration file
 	SystemRegistriesConfPath string
-	// If not "", overrides the default path for the authentication file
+	// If not "", overrides the default path for the authentication file, but only new format files
 	AuthFilePath string
+	// if not "", overrides the default path for the authentication file, but with the legacy format
+	LegacyFormatAuthFilePath string
```

Comment:
This should document that the value is
- only used for reading (if that’s how the code ends up being structured)
- ignored if `AuthFilePath` is set (or whatever the behavior ends up being)

Reply:
part of next push